packages/gutenpride-template/src/readme.txt.mustache

```diff
@@ -0,0 +1,57 @@
+=== Gutenpride ===
```

Comment:
We should rather use the original template that uses more variables. If we don't like some defaults we can override them in the `index.js` file.